### ISSUE TITLE
tutorial VES inversion for a blocky model not working well

### ISSUE THREAD

**Issue by oscarfasanchez** [gimli-org/gimli#393]
hello, I want to report an issue
### Problem description

Please describe your issue here.

### Your environment

**Operating system**:  Windows
**Python version**: 3.8.13
**pyGIMLi version**: 1.2.5
**Way of installation**: Conda package

### Steps to reproduce

I downloaded it from here:
https://www.pygimli.org/_tutorials_auto/3_inversion/plot_2-dc1dblock.html#sphx-glr-tutorials-auto-3-inversion-plot-2-dc1dblock-py

in the last cell:
```
fig, ax = plt.subplots(ncols=2, figsize=(8, 6))  # two-column figure
drawModel1D(ax[0], synthk, synres, plot='semilogx', color='r')
drawModel1D(ax[0], thk, res, color='b')
ax[0].grid(True, which='both')
ax[0].set_ylabel('z (m)')
ax[0].set_xlabel(r'$\rho$ ($\Omega$m)')
ax[1].loglog(rhoa, ab2, 'rx-', label='data')  # sounding curve
ax[1].loglog(inv.response(), ab2, 'b-', label='response')
ax[1].set_ylim((max(ab2), min(ab2)))  # downwards according to penetration
ax[1].grid(True, which='both')
ax[1].set_xlabel(r'$\rho_a$ ($\Omega$m)')
ax[1].set_ylabel('AB/2 (m)')
ax[1].legend(loc='best')
plt.show()
```

### Expected behavior

it should show both images

### Actual behavior

![image](https://user-images.githubusercontent.com/39230212/170832537-a2ebabb2-c47b-411c-bdba-8a21297e283b.png)

![image](https://user-images.githubusercontent.com/39230212/170832651-e46a35de-212f-4683-bcb5-a44890cc3899.png)

**Comment by oscarfasanchez** [comment]
I tried to go to an older versión, but I got this error: https://github.com/gimli-org/gimli/issues/391

**Comment by halbmy** [comment]
Thank you for pointing out this problem which is the same as alread reported in #389 

The newest matplotlib version is using `np.newaxis` on all input plot data which wasn't supported by the pyGIMLi Vector. This was fixed in https://github.com/gimli-org/gimli/commit/bcde9459e8391aa78e46dd4f0819a0d4b86524f9 

In the meanwhile, `v.array()` or `np.array(v)` should help (e.g. `ax[1].loglog(rhoa.array(), ab2, ...`).

**Comment by halbmy** [comment]
This should be fixed with pyGIMLi v1.2.6. If the problems persist, don't hesitate to reopen the issue.
